test/controllers/requests_test.rb

```diff
@@ -4,11 +4,25 @@ class RequestsTest < ActionDispatch::IntegrationTest
   setup do
     Peek.adapter.reset
     Peek.reset
+    cookies['peek'] = 'true'
   end
 
-  test "the request id is set" do
+  test "the request id is set when the peek cookie is set" do
     assert_empty Peek.adapter.requests
     get '/'
     assert_not_empty Peek.adapter.requests
```

Comment:
I don't think the test case changed here at all, just need to update the test body:

```ruby
assert_empty Peek.request_id
get '/'
refute_empty Peek.request_id
```